pyopensprinkler/__init__.py

```diff
@@ -257,6 +257,16 @@ def set_water_level(self, level):
 
         return self._set_option("wl", level)
 
+    def run_once_program(self, station_times):
+        """Run once program"""
+        params = {"t": station_times}
```

Comment:
Is it worth to to check the length of `station_times` with the number of stations?

Reply:
@vinteo it doesn't matter actually. For example I tested with my setup which has 24 stations with `controller.run_once_program([0, 60])`. If you wanted to skip stations you do need to explicitly set them, but the tail doesn't make any difference.

Reply:
ok great, makes it easy